include/digitsDFA.h

```diff
@@ -0,0 +1,17 @@
+#ifndef __DIGITSDFA_H__
+#define __DIGITSDFA_H__
+
+#include "dfa.h"
+#include <string>
+
+int error(char c); // used in error state
```

Comment:
Don't need to include this lines 7-9 in header. They won't and shouldn't be used by anyone else.